Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM $PARENT_IMAGE
 ARG PYTORCH_DEPS=cpuonly
 ARG PYTHON_VERSION=3.7
 
+# for tzdata
```

Comment:
I think I did that because it was somehow needed (the image wouldn't build otherwise).
But as Gitlab restricted their CI, I haven't build/tested the image for a while.
I'm also thinking about using [cog](https://github.com/replicate/cog) instead.